ui/src/test/AuditView/audit-hosts.test.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import { mount } from 'enzyme';
```

Comment:
We are moving away from enzyme because we have found that it is not keeping pace with the changes to react.  We are starting the process of adopting react testing, though if we move to a monorepo that might change.

Reply:
Will not change the tests as we will be moving to th emonorepo